app/renderers/user_avatar.rb

```diff
@@ -4,32 +4,52 @@ class UserAvatar
   include ActionView::Helpers::AssetTagHelper
   include ActionView::Helpers::TagHelper
 
-  def initialize(user)
+  class SizeNotDefined < StandardError; end
+
+  AVATAR_SIZES = [:small, :large].freeze
+  LARGE_SIZE = 250
```

Comment:
The name is a bit ambiguous. Maybe we could rename this to be `AVATAR_LARGE_SIZE_PIXELS`?

Reply:
Hmm yeah. Maybe `LARGE_PIXEL_SIZE`? Now that I'm looking at this, I wonder if the `AVATAR_` prefix is redundant, especially from the POV of this namespace. I wonder if these work better like this:
```ruby
UserAvatar::SIZES
UserAvatar::LARGE_PIXEL_SIZE
```